---
title: Web Tier pipeline execution failing during TestCacheInvalidation
description: Description
solution: Experience Manager as a Cloud Service
product: Experience Manager as a Cloud Service
applies-to: Experience Manager as a Cloud Service
keywords: KCS
resolution: Resolution
internal-notes: answer found in https://dmasupport.slack.com/archives/C013SBSHPKK/p1645102872540889?thread_ts=1645102277.855389&cid=C013SBSHPKK
bug: false
article-created-by: Vincent Frey
article-created-date: 3/31/2022 6:42:09 AM
article-published-by: Vincent Frey
article-published-date: 3/31/2022 6:52:55 AM
version-number: 1
article-number: KA-19052
dynamics-url: https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=4a8a30af-bdb0-ec11-9840-0022480bde18
exl-id: 29d6e807-7757-443e-9fdf-57af339f00d7
---
# Web Tier pipeline execution failing during TestCacheInvalidation

## Description


The Web Tier pipeline execution is failing and you observe the message below in the logs:
```
=== RUN TestCacheInvalidation
main_test.go:237: Test fails. Expected header 'X-Vhost' to be existing and not: 'catch-all', got: 'catch-all'
--- FAIL: TestCacheInvalidation (0.00s)
FAIL
exit status 1<br><br>
```


## Resolution


Verify if your [!DNL Dispatcher] configuration has properly configured some vhost for [!DNL Dispatcher] cache invalidation (call being done to host 127.0.0.1) and if you have an X-Vhost header set that is neither empty

 e.g., as in https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/dispatcher.cloud/src/conf.d/available_vhosts/default.vhost#L24
 
 and that the request does not reach catch-all vhost with X-Vhost header set to “catch-all” as in https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/dispatcher.cloud/src/conf.d/dispatcher_vhost.conf#L136
